content/developer/writing_widgets/widget-backend.md

```diff
@@ -21,15 +21,15 @@ Example of working widget with backend can be found [here](https://github.com/cl
 
 ## Defining Endpoints
 
-To create endpoint per widget you need to create `backend.js` file with at least one endpoint definition. That file must be placed in widget main folder similarly to `widget.js` file.
+To create endpoint per widget you need to create `backend.ts` file with at least one endpoint definition. That file must be placed in widget main folder similarly to `widget.js` file.
```

Comment:
Do you think that it is reasonable to change also `widget.js` to `widget.ts`?

Reply:
No, as there is no change in this area.